index.html

```diff
@@ -1,22 +1,15 @@
 <!doctype html>
-<!--
-Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
-This code may only be used under the BSD style license found at https://polymer.github.io/LICENSE
-The complete set of authors may be found at https://polymer.github.io/AUTHORS
-The complete set of contributors may be found at https://polymer.github.io/CONTRIBUTORS
-Code distributed by Google as part of the polymer project is also
-subject to an additional IP rights grant found at https://polymer.github.io/PATENTS
--->
+<!-- Copyright (c) 2015 Google Inc. All rights reserved. -->
 <html>
 <head>
-  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
```

Comment:
I don't now if something changed for the whole GWC project, but I noticed it wasn't removed in the [`<google-sheets>` PR](https://github.com/GoogleWebComponents/google-sheets/pull/18/), so I wanted to ask about it.